src/theory/strings/core_solver.cpp

```diff
@@ -2292,14 +2292,16 @@ bool CoreSolver::processSimpleDeq(std::vector<Node>& nfi,
           << "Disequality normalize empty" << std::endl;
       // the antecedant
       std::vector<Node> ant;
-      // the antecedant that is not explainable in this context
       std::vector<Node> antn;
```

Comment:
I see you deleted the documentation of this field. Maybe you could replace the documentation with something updated?